src/ajax/ajax.js

```diff
@@ -43,10 +44,33 @@ define([ "shoestring" ], function(){
 			return;
 		}
 
+		// create parameter string from data object
+		if( settings.data !== null ){
```

Comment:
`for( x in undefined ) { console.log(x) }' appears to work just fine! Though, it seems like a good idea to just rely on the falsity of `settings.data` as `null` or `undefined` for clarity.